Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_ObjectAutoGrab.cs

```diff
@@ -5,36 +5,38 @@
 
     public class VRTK_ObjectAutoGrab : MonoBehaviour
     {
-        public GameObject objectToGrab;
+        public VRTK_InteractableObject objectToGrab;
         public bool cloneGrabbedObject;
 
         private VRTK_InteractGrab controller;
 
         private IEnumerator Start()
         {
             controller = GetComponent<VRTK_InteractGrab>();
-
             if (!controller)
             {
                 Debug.LogError("The VRTK_InteractGrab script is required to be attached to the controller along with this script.");
+                yield break;
```

Comment:
Do you want to return out of this function here? As far as I know Debug.LogError will stop/pause the scene with the error, but you then in theory at this paused state add the interact grab script to the controller and resume the scene for it to carry on working.

If it's returned at this point then unpausing after the break won't actually carry on if you've applied the fix?